tests/test_port_dpb_system.py

```diff
@@ -248,25 +242,380 @@ def test_port_breakout_with_acl(self, dvs):
+        rootPortName = portGroup[0]
+        breakoutMode2x = "2x50G"
+
+        # Breakout port and expect that newly created ports are
+        # automatically added to VLANs and ACL tables as per
+        # default_config_db.json
+        self.dvs_acl.verify_acl_group_num(0)
+        self.dvs_vlan.get_and_verify_vlan_member_ids(0)
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode1x)
+        dvs.change_port_breakout_mode(rootPortName, breakoutMode4x, breakoutOption)
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode4x)
+        self.dvs_acl.verify_acl_group_num(len(portGroup))
+        self.dvs_vlan.get_and_verify_vlan_member_ids(len(portGroup))
+
+        # Breakout port and expect that Ethernet4 and Ethernet6 remain in
+        # ACL and VLAN where as Ethernet5 and Ethernet7 get removed from
+        # VLAN and ACL table
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode4x)
+        dvs.change_port_breakout_mode(rootPortName, breakoutMode2x, breakoutOption + " -f")
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode2x)
+        self.dvs_acl.verify_acl_group_num(2)
+        self.dvs_vlan.get_and_verify_vlan_member_ids(2)
+
+        # Breakout again and verify that only root port (Ethernet4) remains in
+        # in VLAN and ACL and Ethernet6 gets removed.
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode2x)
+        dvs.change_port_breakout_mode(rootPortName, breakoutMode1x, breakoutOption + " -f")
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode1x)
+        self.dvs_acl.verify_acl_group_num(1)
+        self.dvs_vlan.get_and_verify_vlan_member_ids(1)
+
+        # Breakout port without "-l" option and ensure that root port
+        # gets removed from VLAN and ACL
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode1x)
+        dvs.change_port_breakout_mode(rootPortName, breakoutMode2x, "-f")
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode2x)
+        self.dvs_acl.verify_acl_group_num(0)
+        self.dvs_vlan.get_and_verify_vlan_member_ids(0)
+
+        #Cleanup
+
+        # Move both Ethernet0 and Ethernet4 back to default mode
+        dvs.change_port_breakout_mode("Ethernet0", breakoutMode1x)
+        dpb.verify_port_breakout_mode(dvs, "Ethernet0", breakoutMode1x)
+        dvs.change_port_breakout_mode("Ethernet4", breakoutMode1x)
+        dpb.verify_port_breakout_mode(dvs, "Ethernet4", breakoutMode1x)
+
+        # Delete VLANs and ACL tables
+        bind_ports = []
+        for aclTableName in aclTableNames:
+            self.dvs_acl.remove_acl_table(aclTableName)
+        for vlanID in vlanIDs:
+            self.dvs_vlan.remove_vlan(vlanID)
+
+        # Verify cleanup
+        self.dvs_acl.verify_acl_table_count(0)
+        self.dvs_vlan.get_and_verify_vlan_ids(0)
+
+        ##### Interface dependency test ############
```

Comment:
I notice that, each time INTERFACE is tested independently, not with ACL, VLAN. Any reason for that?. May be in one test, we can keep all.

Reply:
Because, Ethernet0 is treated as  L2 port and Ethernet8 as L3 only.